src/contrib/monitoring/check_zookeeper.py

```diff
@@ -251,6 +256,13 @@ def _parse_stat(self, data):
                 result['zk_znode_count'] = int(m.group(1))
                 continue
 
+            m = re.match('Zxid: (0x[0-9a-fA-F]+)', line)
+            if m is not None:
+                result['zk_zxid']         = int(m.group(1), 16) # convert from hex
```

Comment:
Since the zxid is being split up below, it may be valuable to leave this in hex.